src/integration/tests/firstTest.test.jsx

```diff
@@ -68,5 +66,29 @@ describe("Testes", () => {
     expect(screen.queryByTestId(`TASK_CONTAINER_${taskMsg}`)).not.toBeInTheDocument();
   });
 
+  it('validate if click button info it will show the back button and the title with task name.', () => {
```

Comment:
Name suggestion for the scenario

```suggestion
  it('Patient should be able to open task details', () => {
```